src/Controller/Wysiwyg/StylesetController.php

```diff
@@ -15,10 +17,11 @@ public function __construct(WysiwygStylesSetService $wysiwygStylesSetService)
         $this->wysiwygStylesSetService = $wysiwygStylesSetService;
     }
 
-    public function iframe(string $name): Response
+    public function iframe(string $name, string $language): Response
     {
         return $this->render('@EMSCore/wysiwyg_styles_set/iframe.html.twig', [
             'styleSet' => $this->wysiwygStylesSetService->getByName($name),
+            'language' => \explode('_', $language)[0],
```

Comment:
```suggestion
            'language' => \array_shift(\explode('_', $language)),
```

Reply:
This is not really correct can not array_shift on the return of an explode but will fix!